examples/redis/Examples.StackExchangeRedis/README.md

```diff
@@ -0,0 +1,5 @@
+# StackExchangeRedis Instrumentation for OpenTelemetry .NET - Example
+
+An example application that shows how to use
```

Comment:
one line about the manually created activity, inside which Rediscall is made.. to demo the auto-parenting capabilities.. something of that line.. or simply remove the manual activity creation completely.